Lib/pdb.py

```diff
@@ -590,6 +591,36 @@ def default(self, line):
         except:
             self._error_exc()
 
+    def _replace_convenience_variable(self, line):
```

Comment:
Should the function name (and docstring) be "variables" plural, since it replaces all occurrences?